app/next-client-app/components/jobs/JobUtils.tsx

```diff
@@ -1,21 +1,19 @@
 export const FindGeneralStatus = (jobsData: Job[]) => {
-  let generalStatus = "NOT_STARTED";
-  if (jobsData.length > 0) {
-    if (
-      jobsData?.some((job) => job.status && job.status.value === "IN_PROGRESS")
-    ) {
-      generalStatus = "IN_PROGRESS";
-    } else if (
-      jobsData?.some((job) => job.status && job.status.value === "FAILED")
-    ) {
-      generalStatus = "FAILED";
-    } else if (
-      jobsData?.every((job) => job.status && job.status.value === "COMPLETE")
-    ) {
-      generalStatus = "COMPLETE";
-    }
+  if (
+    jobsData.some((job) => job.status && job.status.value === "IN_PROGRESS")
+  ) {
+    return "IN_PROGRESS";
```

Comment:
Happy to approve to get this fix in - but looks like we have configuration masking as constants, and not benefiting from any typing. 

As in - lots of strings being passed about when we'd really like types.